zenml/components/bulk_inferrer/executor.py

```diff
@@ -14,49 +14,33 @@
 """ZenML bulk inferrer executor."""
```

Comment:
Im not quite sure about the executor of the BulkInferrer anymore... Is it being used in the BatchInferencePipeline?